RELEASE-NOTES.md

```diff
@@ -15,6 +15,11 @@
 [#4720](https://github.com/pymc-devs/pymc3/pull/4720)).
 + `plot_gp_dist` is now available at the package level: `pm.plot_gp_dist` ([see #4835](https://github.com/pymc-devs/pymc3/pull/4835)).
 + The experimental JAX sampling was removed. It will be included in the upcoming `v4` release ([see #4866](https://github.com/pymc-devs/pymc3/pull/4866)).
++ `pm.intX` no longer downcasts integers unnecessarily (see [#4569](https://github.com/pymc-devs/pymc3/pull/4569))
```

Comment:
We should always put `New Features` at the top.

Reply:
Good idea. Can you add a line saying the same in Release Checklist - https://github.com/pymc-devs/pymc3/wiki/Release-Checklist?